addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -21,79 +21,42 @@
             padding: 0!important;
         }
         background-color: $o-view-background-color;
```

Comment:
Consider acting on the bootstrap variable; currently it is `$table-bg: null !default;`, we may set it to `$o-view-background-color`instead